L1-FINGERPRINT_Toolbox_Overview.adoc

```diff
@@ -15,131 +15,133 @@ A cast is the PAI, the individual artefact that will be presented to the sensor
-
-==== Presentation Practice
-Presentation practice will help to minimize noise in the provided sample. This noise is related to poor presentation of the finger or PAI to the sensor (or mold), generating a poor result. For example, sliding your finger on the sensor instead of holding it still will cause the sample to be "smeared". Each type of PAI will have its own unique requirements for producing the highest quality result and should be practiced individually.
-
-=== Flat PAI data acquisition and processing
-
-Fingerprint image acquisition for flat PAIs is the same for all attacks. An image of the subject’s finger is captured using a high quality commercial fingerprint scanner. The image can be from a slap or rolled image. If a rolled image is acquired, the evaluator may want to crop the fingerprint image to make a smaller mold containing only the central part of the fingerprint that would usually be in contact with the TOE sensor during normal operation. This region should be large enough to cover as much of the sensor as possible when presenting the PAI to the TOE. This same finger shall be used to enroll the subject into the TOE. Each image shall be acquired under controlled and favorable conditions so as to produce high quality friction ridge information. When supported by the scanner, the image file received shall be in bitmap, TIFF, RAW, or other lossless file formats. Image files using the WSQ format at a bitrate of 2.25 (≅5:1 compression) are acceptable if lossless file formats are not available from the scanner.
-
-The image then needs to be processed such that the resultant image contains only black (0 in the 8-bit grayscale color space) and white (255 in the 8-bit grayscale color space) pixels. This can be done by hand using photo editing software suitable for pixel level adjustments or in an automated fashion using binarization software. The resulting enhanced images should have black ridges and white valleys. Pores and creases are also white. The resultant image should be life-sized and remain at the highest resolution available. Resizing the image shall not use interpolation without justification approved by the BIT.
+The image (whether from the scanner or captured from the latent print) then needs to be processed such that the resultant image contains only black (0 in the 8-bit grayscale color space) and white (255 in the 8-bit grayscale color space) pixels. This can be done by hand using photo editing software suitable for pixel level adjustments or in an automated fashion using binarization software. The resulting enhanced images should have black ridges and white valleys. Pores and creases are also white. The resultant image should be life-sized and remain at the highest resolution available. Resizing the image shall not use interpolation without justification approved by the BIT.
```

Comment:
May be worded better, but trying to double-check the lift for quality after binarization.

```suggestion
The image (whether from the scanner or captured from the latent print) then needs to be processed such that the resultant image contains only black (0 in the 8-bit grayscale color space) and white (255 in the 8-bit grayscale color space) pixels. This can be done by hand using photo editing software suitable for pixel level adjustments or in an automated fashion using binarization software. The resulting enhanced images should have black ridges and white valleys. Pores and creases are also white. The resultant image should be life-sized and remain at the highest resolution available. Resizing the image shall not use interpolation without justification approved by the BIT.

If excessive powder or background artifacts involved in the acquisition of the fingermark create excessive false ridge detail in the resulting binarized image, the steps for acquiring a latent print shall be repeated.
```